docs/notebooks/cold_plasma_tensor_elements.ipynb

```diff
@@ -49,248 +58,536 @@
    ]
   },
   {
-   "cell_type": "raw",
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
+   "source": [
+    "[astropy.units]: https://docs.astropy.org/en/stable/units/index.html\n",
+    "[cold_plasma_permittivity_SDP]: https://docs.plasmapy.org/en/stable/api/plasmapy.formulary.dielectric.cold_plasma_permittivity_SDP.html\n",
+    "[cold_plasma_permittivity_LRP]: https://docs.plasmapy.org/en/stable/api/plasmapy.formulary.dielectric.cold_plasma_permittivity_LRP.html\n",
+    "\n",
+    "For more information, check out the documentation on [cold_plasma_permittivity_SDP] and [cold_plasma_permittivity_LRP].\n",
+    "\n",
+    "Let's use [astropy.units] to define some parameters, such as the magnetic field magnitude, plasma species, and densities."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {
-    "raw_mimetype": "text/restructuredtext"
+    "pycharm": {
+     "name": "#%%\n"
+    }
    },
+   "outputs": [],
    "source": [
-    "Take a look at the docs to :func:`~plasmapy.formulary.dielectric.cold_plasma_permittivity_SDP` and :func:`~plasmapy.formulary.dielectric.cold_plasma_permittivity_LRP` for more information on these."
+    "B = 2 * u.T\n",
+    "species = [\"e\", \"D+\"]\n",
+    "n = [1e18 * u.m**-3, 1e18 * u.m**-3]"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
    "source": [
-    "Let's define some parameters, such as the magnetic field magnitude,\n",
-    "the plasma species and densities and the frequency band of interest\n",
-    "\n"
+    "Let's pick some frequencies in the radio frequency range."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
-    "jupyter": {
-     "outputs_hidden": false
+    "pycharm": {
+     "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
-    "B = 2 * u.T\n",
-    "species = [\"e\", \"D+\"]\n",
-    "n = [1e18 * u.m**-3, 1e18 * u.m**-3]\n",
+    "f_min = 1 * u.MHz\n",
+    "f_max = 200 * u.GHz\n",
+    "\n",
+    "f = np.geomspace(f_min, f_max, num=5000).to(u.Hz)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% raw\n"
+    }
+   },
+   "source": [
+    "[equivalency]: https://docs.astropy.org/en/stable/units/equivalencies.html\n",
+    "[angular frequencies]: https://docs.plasmapy.org/en/stable/development/code_guide.html?highlight=angular%20frequencies#angular-frequencies\n",
```

Comment:
```suggestion
    "[angular frequencies]: https://docs.plasmapy.org/en/stable/development/code_guide.html#angular-frequencies\n",
```

[optional]  Both links work, but this one will go directly to the section header without highlighting the words "angular" and "frequencies" on the whole page.